tracim/tracim
# Feat/3411  add span wrapping mention

#3411 

## Checkpoints

These points must be checked before merging. Please don't edit them out.

**For developers**



**For code reviewers**


**For testers**


To test:
- open your browser console (ctrl + shift + I - Console)
- make mentions at comments and html documents
- see if you have a `mention.created` TLM
